packages/fluentui/react-northstar/src/themes/teams/getPointerStyles.ts

```diff
@@ -173,29 +173,29 @@ export const getPointerStyles = (options: GetPointerStylesOptions): ICSSInJSStyl
 
           left: gap,
           bottom: `calc(${width} - ${height} + ${borderSize})`,
-          transform: 'rotate(90deg) /* @noflip */',
+          transform: `rotate(${rtl ? -90 : 90}deg)`,
         }),
         ...(placement === 'top' && {
           height: `calc(${width} + (${gap} * 2))`,
           width: height,
 
           left: gap,
           bottom: `calc(${gap} + ${height} - ${borderSize})`,
-          transform: 'rotate(-90deg) /* @noflip */',
+          transform: `rotate(${rtl ? 90 : -90}deg)`,
         }),
         ...(placement === 'left' && {
           height: width,
           width: height,
 
           left: 0,
-          transform: 'rotate(180deg) /* @noflip */',
+          transform: `rotate(${rtl ? 0 : 180}deg)`,
```

Comment:
For context:
- `cssjanus` used in Emotion does not transform [`rotate()`](https://cssjanus.github.io/#input/.a%20%7B%20transform%3A%20rotate(0deg)%3B%20%7D), while `rtl-css-js` used in Fela plugin - [does](https://github.com/kentcdodds/rtl-css-js/blob/66582a73c1f80028f53a8ffa430dfdeb472ed6d0/src/__tests__/index.js#L279)
- to make it compatible we should not rely on flipping behavior

---

```suggestion
          transform: rtl ? 'rotate(0) /* @noflip */' : 'rotate(180deg) /* @noflip */',
```

Let's keep it compatible with Emotion rather than reverting changes: https://codesandbox.io/s/amazing-resonance-pxt1q?file=/src/App.js

(it's interesting that `rtl-css-js` does not actually flip `0` and `180deg`)

Reply:
Done